.github/workflows/build.yml

```diff
@@ -70,7 +70,7 @@ jobs:
       - name: 'Install xerces, qt5, and qwt (macOS)'
         if: runner.os == 'macOS'
         run: |-
-          brew install xerces-c qt5 qwt-qt5
+          brew install xerces-c qt@5 qwt-qt5
```

Comment:
Also change this line below
`CMAKE_ARGS+=" -DQt5_DIR=$(brew --prefix qt5)/lib/cmake/Qt5"`